src/resources.ts

```diff
@@ -7,5 +7,6 @@ export const resources = {
     PADDINGBOTTOMDOWN: 0,
     PADDINGTOPDOWN: 10,
     PADDINGCATEGORYLEFT: 0,
-    PADDINGPERCENTAGEDOWN: 0
+    PADDINGPERCENTAGEDOWN: 0,
+    LINEWIGHT: 3

```

Comment:
Great that you added the line weight in the resources file, but there's a typo

Reply:
just changed!